api/src/test/java/org/pmiops/workbench/api/CohortReviewControllerTest.java

```diff
@@ -233,6 +248,10 @@ public void setUp() {
             .conceptId(String.valueOf(TestDemo.WHITE.conceptId))
             .name(TestDemo.WHITE.name));
 
+    genderRaceEthnicityMap =
```

Comment:
You can make a static `asMap` method inside your enum that returns this map.

Reply:
Done.